src/book.py

```diff
@@ -627,11 +627,160 @@ def _read_bitpanda_pro_trades(self, file_path: Path) -> None:
+            "buy": "Buy",
+            "sell": "Sell",
+        }
+
+        with open(file_path, encoding="utf8") as f:
+            reader = csv.reader(f)
+
+            # skip header, there are multiple lines
+            # the last one is the actual header
+            line = next(reader)
+            line = next(reader)
+            line = next(reader)
+            line = next(reader)
+            line = next(reader)
+            line = next(reader)
+            line = next(reader)
+
+            if line != [
+                "Transaction ID",
+                "Timestamp",
+                "Transaction Type",
+                "In/Out",
+                "Amount Fiat",
+                "Fiat",
+                "Amount Asset",
+                "Asset",
+                "Asset market price",
+                "Asset market price currency",
+                "Asset class",
+                "Product ID",
+                "Fee",
+                "Fee asset",
+                "Spread",
+                "Spread Currency",
+            ]:
+                raise RuntimeError(f"Expected header did not match {line}")
+
+            for (
+                _tx_id,
+                csv_utc_time,
+                operation,
+                _inout,
+                amount_fiat,
+                fiat,
+                amount_asset,
+                asset,
+                asset_price,
+                asset_price_currency,
+                asset_class,
+                _product_id,
+                fee,
+                fee_currency,
+                _spread,
+                _spread_currency,
+            ) in reader:
+                row = reader.line_num
+
+                # make RFC3339 timestamp ISO 8601 parseable
+                if csv_utc_time[-1] == "Z":
+                    csv_utc_time = csv_utc_time[:-1] + "+00:00"
+
+                # timezone information is already taken care of with this
+                utc_time = datetime.datetime.fromisoformat(csv_utc_time)
+
+                # transfer ops seem to be akin to airdrops. In my case I got a
+                # CocaCola transfer, which I don't want to track. Would need to
+                # be implemented if need be.
+                if operation == "transfer":
+                    log.warning(
+                        f"'Transfer' operations are not "
+                        f"implemented, skipping: {file_path} line {row}."
+                    )
+                    continue
+
+                # fail for unknown ops
+                try:
+                    operation = operation_mapping[operation]
+                except KeyError:
+                    raise RuntimeError(f"Unsupported operation '{operation}'")
+
+                if operation == "Deposit":
+                    # This identifies a fiat deposit. Error on yet unknown deposits,
+                    # they will need to be implemented separately
+                    if fiat != "EUR" and asset != "EUR" and asset_class != "fiat":
```

Comment:
Why is this important to distinguish? Deposits doesnt matter for taxation stuff. They are only there for information.

Reply:
I see, my reasoning is that if I don't know the full details of the deposit as a spec, it can be "anything", other currencies, "airdrops" (?) and thus it should fail hard immediately as it's unexpected. But if you say its unreasonable I am open to just remove it.

Reply:
It is also possible to have coins deposited (transfered from somewhere else aka withdrawn from somewhere and deposited here). This is also not taxable. 

It would be a huge problem if they declare airdrops as deposits and I really hope that they don't do that...
If they declare airdrops as deposit

Reply:
I think we can remove this check completely, as it shouldn't matter which fiat currency it is. I just tried replacing EUR with USD in my CSV (USDT for fee asset), and everything still worked as intended. My proposed changes are here: https://github.com/sebastianelsner/CoinTaxman/pull/1